app/models/lesson.rb

```diff
@@ -56,6 +56,10 @@ def has_update_warning?
     !update_warning.blank?
   end
 
+  def has_teacher_notes?
+    !teacher_notes.blank?
```

Comment:
@mgoren Here's a cool Rails/ActiveRecord trick that will allow us to remove this method and its tests entirely (and the other similar methods in this model eventually).

Since `teacher_notes` is an ActiveRecord attribute, we can just call `teacher_notes?` which acts as a boolean method.